generators/app/index.js

```diff
@@ -1,39 +1,41 @@
-'use strict';
+"use strict";
 
-const Generator = require('yeoman-generator'),
-PromptHandler = require('./resources/handler'),
-handler = new PromptHandler();
+const Generator = require("yeoman-generator"),
+  PromptHandler = require("./resources/handler"),
+  handler = new PromptHandler();
 
 module.exports = Generator.extend({
-
   // Configurations will be loaded here.
   // Ask for user input
   prompting() {
-    return handler.initialPrompt.call(this)
-    .then(answers => handler.secondPrompt.call(this, answers))
-    .then(answers => handler.setPlugins.call(this, answers))
-    .then(answers => handler.thirdPrompt.call(this, answers))
-    .then(answers => handler.finishPrompts.call(this,answers))
-    .then(answers => handler.setNewDestinationRoot.call(this,answers.dir));
+    return handler.initialPrompt
+      .call(this)
+      .then(answers => handler.secondPrompt.call(this, answers))
+      .then(answers => handler.setPlugins.call(this, answers))
+      .then(answers => handler.thirdPrompt.call(this, answers))
+      .then(answers => handler.finishPrompts.call(this, answers))
+      .then(answers => handler.setNewDestinationRoot.call(this, answers.dir));
   },
 
   // Writing Logic here
   writing: {
     // Copy the configuration files
-    config: function () { 
+    config: function() {
       handler.creatingConfigFiles.call(this);
     },
 
     // Copy application files
-    app: function () {
+    app: function() {
       handler.creatingAppFiles.call(this);
       handler.removingUnnecessaryFiles.call(this);
     },
 
     // Install Dependencies if wanted
-    install: function () {
-      if(this.props.dependencies){
-        this.installDependencies();  
+    install: function() {
+      if (this.props.dependencies) {
+        // REMOVE: 
```

Comment:
what's this?

Reply:
Something I need to do later, for some reason to install dependencies for plugins we're asking if want to or not, in my opinion this shouldn't be asked and generator should just add them 'cause if user select them I cannot think a reason why they wont be installed, I have added a `issue` on github and will remove it.